Site/SiteApplication.php

```diff
@@ -398,6 +398,13 @@ protected function configureErrorHandling(SiteConfigModule $config)
 				$config->errors->base_uri,
 				$config->errors->unix_group));
 
+		if (isset($config->sentry->dsn)) {
+			// Default breadcrumb handlers override the error_reporting settings, so we disable them
+			$client = new Raven_Client($config->sentry->dsn, array('install_default_breadcrumb_handlers' => false));
+			SwatException::addLogger(new SiteSentryExceptionLogger($client));
+			SwatError::addLogger(new SiteSentryErrorLogger($client));
```

Comment:
They could be, but it has the same effect either way. I left them as `setLogger` because I thought it would be good for backwards compatibility, in case somehow Site was updated without the newest Swat code. I'm nevertheless happy to change it to `addLogger` if you think it is more fitting.